### PR TITLE
adding password-less authentication example

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
this helps to login using session key

### CODE REVIEW

File: omero/developers/Web/PublicData.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Public data in OMERO.web
 ========================
 
+
+Public user
+-----------
+
+
```
Comment (mtbc) [inline omero/developers/Web/PublicData.txt]: remove one of the blank lines

File: omero/developers/Web/PublicData.txt [hunk 2]
```diff
@@ -82,3 +87,18 @@ Here is how to go about setting this up on your OMERO.web install:
     can always access the login page using the following link
     `\http://your_host/webclient/login/`.
 
+
+Reusing omero session
```
Comment (mtbc) [inline omero/developers/Web/PublicData.txt]: an "omero session" should be all lower-case?

File: omero/developers/Web/PublicData.txt [hunk 3]
```diff
@@ -82,3 +87,18 @@ Here is how to go about setting this up on your OMERO.web install:
...
+---------------------
+
+Since OMERO 4.4, the OMERO.web framework has supported password-less, omero
+session key-based authentication. For example:
+
+    ::
+
+        http://your_host/webgateway/img_detail/IMAGE_ID/?server=SERVER_ID&bsession=omerosessionkey
```
Comment (mtbc) [inline omero/developers/Web/PublicData.txt]: I assume the parameter really is "bsession". :smiley:

File: omero/developers/Web/PublicData.txt [hunk 4]
```diff
@@ -82,3 +87,18 @@ Here is how to go about setting this up on your OMERO.web install:
...
+
+
+For more details about how to create omero session see
```
Comment (mtbc) [inline omero/developers/Web/PublicData.txt]: "*an* omero session"

File: omero/developers/Web/PublicData.txt (hunk 3 above)
Comment (aleksandra-tarkowska) [inline omero/developers/Web/PublicData.txt]: YES!

**Comment by hflynn** [comment]
you happy with this too @will-moore ?

**Comment by manics** [comment]
Since this is passing a session key can you change the example URL to https?

**Comment by will-moore** [comment]
I haven't tried this myself.
We should probably say what SERVER_ID is.

**Comment by aleksandra-tarkowska** [comment]
is that better now?

**Comment by manics** [comment]
I can't get this to work:

```
$ bin/omero login -s trout.openmicroscopy.org
Username: [user-1]
Password:
Created session cd0ef159-d4b1-42e0-9653-46df919b3368 (user-1@trout.openmicroscopy.org:4064). Idle timeout: 10.0 min. Current group: private-1
```
None of these work:

https://trout.openmicroscopy.org/merge/webclient/img_detail/4598/?server=0&bsession=cd0ef159-d4b1-42e0-9653-46df919b3368
https://trout.openmicroscopy.org/merge/webclient/img_detail/4598/?server=1&bsession=cd0ef159-d4b1-42e0-9653-46df919b3368
https://trout.openmicroscopy.org/merge/webclient/img_detail/4598/?server=merge&bsession=cd0ef159-d4b1-42e0-9653-46df919b3368

**Comment by manics** [comment]
And for no obvious reason it's suddenly started working (with `server=1`)

**Comment by aleksandra-tarkowska** [comment]
@manics Index starts from 1, and it has to be number. Middle link is only one valid

**Comment by will-moore** [comment]
I haven't managed to get a session ID yet to test this, but given the latest update it looks good to me.

**Comment by will-moore** [comment]
Sorry - One final thought. Is "Public Data" the best page to put this? It seems it would only be used if the data were not public.

**Comment by aleksandra-tarkowska** [comment]
@will-moore Public data is not ideal but we will move it to the **Common repository** when we create new section. This is kind of public data, as long as you know the link and have valid session key you can share it with everyone. I'd keep it as it is at the moment

**Comment by will-moore** [comment]
Ok, fine by me.

**Comment by hflynn** [comment]
you happy with this as well now @manics ?

**Comment by manics** [comment]
Yes

**Comment by jburel** [comment]
Do you want to mention that you can use the session id to connect via insight as a note?

**Comment by aleksandra-tarkowska** [comment]
--rebased-to #1165

**Pull request merged**